bindings/python/pymongoarrow/schema.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import collections.abc as abc
+
+from pymongoarrow.types import _normalize_typeid
+
+
+class Schema:
```

Comment:
Can you add some tests for this class? It's a little hard to understand what schemas are supported.

Reply:
Done.